codefellowship/src/main/java/com/shaybrow/codefellowship/constructors/Post.java

```diff
@@ -0,0 +1,23 @@
+package com.shaybrow.codefellowship.constructors;
+
+import javax.persistence.*;
+
+@Entity
+public class Post {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    long id;
+
+
+    @OneToOne

```

Comment:
Here you are saying there is a one to one relationship, but the relationship is just with a string? This is broken code, there is no relationship with String username. Try changing this to the User.

Reply:
Agreed. I didn't really understand what the relationship was supposed to be at that point. This pr has been updated with lab 17 and some of lab 18